libcxx/test/std/numerics/c.math/signbit.pass.cpp

```diff
@@ -0,0 +1,78 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// bool signbit(floating-point-type x); // constexpr since C++23
+
+// We don't control the implementation on windows
+// UNSUPPORTED: windows
+
+// These compilers don't support constexpr `__builtin_signbit` yet.
+// UNSUPPORTED: clang-17, clang-18, clang-19, apple-clang-16, apple-clang-17, apple-clang-18
```

Comment:
There are no AppleClang 17 or 18. Did you mean `apple-clang-15, apple-clang-16`?

Reply:
Thanks for pointing out! Switching to `apple-clang-15` solved my issue. Looking at the [buildkite pipeline configs](https://github.com/llvm/llvm-project/blob/main/libcxx/utils/ci/buildkite-pipeline.yml#L24), LLVM 18 seems to be used on `Apple back-deployment macosx10.{13,15}` and `Apple back-deployment with hardening enabled`. How to know that this is related to `apple-clang-15`?

Reply:
AppleClang is unrelated to the LLVM header version. We currently document AppleClang15 as the only supported AppleClang: https://libcxx.llvm.org/#platform-and-compiler-support

Reply:
Got it. Thanks. In this case, wouldn't it be better to remove `apple-clang-16` from the list of unsupported compiler?

Reply:
No. While we don't officially support it right now we actually use it in the GitHub CI to allow landing some important patches. We'll officially switch over soon too.